test/parallel/test-internal-errors.js

```diff
@@ -341,6 +341,13 @@ assert.strictEqual(
   );
 }
 
+// Test ERR_SCRIPT_EXECUTION_INTERRUPTED
+assert.strictEqual(
```

Comment:
This test does not look particularly useful since the error message formatter does not take any parameters for this error.